src/Diagnostics.ModelsAndUtils/Models/ResponseExtensions/DetectorStatus.cs

```diff
@@ -19,11 +19,11 @@ public static class ResponseDetectorStatusExtensions
         /// <code>
         /// public async static Task<![CDATA[<Response>]]> Run(DataProviders dp, OperationContext cxt, Response res)
         /// {
-        ///     res.SetDetectorStatus(DetectorStatus.Critical, "This is an error message that will come from the detector");
+        ///     res.SetDetectorStatus(InsightStatus.Critical, "This is an error message that will come from the detector");
         /// }
         /// </code>
         /// </example>
-        public static void SetDetectorStatus(this Response response, DetectorStatus status, string message = null)
+        public static void SetDetectorStatus(this Response response, InsightStatus status, string message = null)
```

Comment:
I guess, this method is not getting used in developers? it'll be a breaking change

Reply:
It is not being used anywhere.